### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by bug-smith**
completed.

https://github.com/bugass/Cooking-Curiosity/assets/102183396/5f73900f-1027-423b-a55f-593e48eea48e

### CODE REVIEW

File: css/layout.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+.flex {
+  display: flex;
+}
+
+/* font-family: 'Judson',
+serif;
+font-family: 'Just Me Again Down Here',
+cursive; */
```
Comment (shawnkost) [inline css/layout.css]: Delete all old code

File: index.html [hunk 2]
```diff
@@ -4,17 +4,129 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>AJAX Project</title>
```
Comment (shawnkost) [inline index.html]: Update this `title` to be the name of your project

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+const $breakfastUl = document.querySelector('#breakfast-list');
+const $lunchUl = document.querySelector('#lunch-list');
+const $dinnerUl = document.querySelector('#dinner-list');
+const $breakfastImage = document.querySelector('#breakfast-image');
+const $lunchImage = document.querySelector('#lunch-image');
+const $dinnerImage = document.querySelector('#dinner-image');
+const $breakfastText = document.querySelector('#breakfast-text');
+const $lunchText = document.querySelector('#lunch-text');
+const $dinnerText = document.querySelector('#dinner-text');
+// data.breakfastEntries.push({breakfastImage: xhrB.recipes.image, breakfastIngredients: )
```
Comment (shawnkost) [inline js/main.js]: Delete all old code

File: js/main.js [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+
+const obj = {
+  breakfastData: [],
+  lunchData: [],
+  dinnerData: [],
+  breakfastPreviewCounter: 0,
+  lunchPreviewCounter: 0,
+  dinnerPreviewCounter: 0
+};
```
Comment (shawnkost) [inline js/main.js:28]: Do you need this & your `data` object? What's the difference between the two? It doesn't seem like you're using your `data` object anywhere

File: js/main.js [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+
+function getBreakfastData() {
+  const xhrB = new XMLHttpRequest();
```
Comment (shawnkost) [inline js/main.js]: I would change this to just `xhr`

File: js/main.js [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+  xhrB.addEventListener('load', function () {
+    obj.breakfastData = xhrB.response.recipes[0];
+    $breakfastImage.setAttribute('src', obj.breakfastData.image);
+    breakfastIngredientsLoop();
+  });
+  xhrB.send();
+}
+
+function getLunchData() {
+  const xhrL = new XMLHttpRequest();
```
Comment (shawnkost) [inline js/main.js]: I would change this to just `xhr`

File: js/main.js [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+  xhrL.addEventListener('load', function () {
+    obj.lunchData = xhrL.response.recipes[0];
+    $lunchImage.setAttribute('src', obj.lunchData.image);
+    lunchIngredientsLoop();
+  });
+  xhrL.send();
+}
+
+function getDinnerData() {
+  const xhrD = new XMLHttpRequest();
```
Comment (shawnkost) [inline js/main.js]: I would change this to just `xhr`

File: js/main.js [hunk 8]
```diff
@@ -0,0 +1,174 @@
...
+//   //   <div>
+//   //     <p id="breakfast-info">Toast the sesame seeds, about 350 degrees in the oven for about 10-15 minutes. Keep an eye on them to make sure they
+//   //       don't burn.Mix together the following to make the dressing: olive oil, vinegar, sugar, salt, pepper, green onions,
+//   //       chicken flavor packet from the ramen noodle package.Crush the ramen noodles until there are no large chunks (small
+//   //       chunks are OK).Combine the shredded cabbage and ramen noodles in a large bowl.Pour the dressing on the cabbage/noodle
+//   //       mixture and toss to coat.Top with the toasted sesame seeds and almonds.</p>
+//   //   </div>
+//   // </div>
+
+// }
```
Comment (shawnkost) [inline js/main.js]: Delete all old code

File: js/main.js [hunk 9]
```diff
@@ -0,0 +1,174 @@
...
+  $dinnerColumn.classList.toggle('hidden');
+  $lunchRow.classList.toggle('hidden');
+  $breakfastRow.classList.toggle('hidden');
+  obj.dinnerPreviewCounter++;
+  if (obj.dinnerPreviewCounter % 2 === 0) { $dinnerText.textContent = 'DINNER'; } else { $dinnerText.textContent = obj.dinnerData.title; }
+});
+
+// function renderImage(
+
+// )
```
Comment (shawnkost) [inline js/main.js]: Delete all old code

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looking good so far Bug! Just a few changes here

File: js/main.js (hunk 4 above)
Comment (bug-smith) [reply js/main.js:28]: I am not using it yet, was moreso setting up for local storage, to determine how many, and what recipes are saved. please advise.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**